run_kraken.py

```diff
@@ -113,7 +114,7 @@ def main(cfg):
                 "Using port 8081 as %s isn't a valid port number" % (port)
             )
             port = 8081
```

Comment:
think you need to not set this here and have a get of the port above from the config

Reply:
@paigerube14 tweaked it to exit(1) if the port passed by the user in the config is invalid. Also removed the defaults to force the user to know what address and port is being used. Thoughts?